scripts/restore

```diff
@@ -69,6 +69,14 @@ ynh_restore_file --origin_path="/etc/logrotate.d/$app"
 #=================================================
 # RESTORE THE FAIL2BAN CONFIGURATION
 #=================================================
+ynh_script_progression --message="Restoring fail2ban configuration..."
+
+ynh_restore_file --origin_path="/var/log/$domain-access.log"
+# If it doesn't exist, create the logfile, required before configuring fail2ban
+log4fail2ban="/var/log/nginx/${domain}-access.log"
+if [ ! -f "${log4fail2ban}" ]; then
```

Comment:
```suggestion
if [ ! -f "${log4fail2ban:-}" ]; then
```

Reply:
What is the point of `:-` ?
I don't know this `pattern`

Reply:
cf bash man page -> https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
Ok, thank's